cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -1202,7 +1202,8 @@ macro(build_thrift)
   endif()
 
   if("${THRIFT_SOURCE_URL}" STREQUAL "FROM-APACHE-MIRROR")
-    get_apache_mirror()
+    #     get_apache_mirror()
+    set(APACHE_MIRROR http://apache.osuosl.org/)
```

Comment:
Oof. Can we modify the script to fall back to a known mirror if there is an SSL error?

Reply:
Good idea, will do.